books/models.py

```diff
@@ -996,7 +998,8 @@ def books(self):
                     'urls': book.book_urls(),
                     'last_updated_pdf': book.last_updated_pdf,
                     'has_faculty_resources': has_faculty_resources,
-                    'has_student_resources': has_student_resources
+                    'has_student_resources': has_student_resources,
+                    'errata_content': book.errata_content()
```

Comment:
does this need to be on the book index page? I think it's only needed on the book detail page

Reply:
I should have removed this and missed it. Thanks for catching.